library/std/src/path.rs

```diff
@@ -2405,6 +2418,18 @@ impl Path {
     ///
     /// This is a convenience function that coerces errors to false. If you want to
     /// check errors, call [`fs::metadata`].
+    ///
+    /// An example of proper error handling.
```

Comment:
Would it perhaps make more sense to just link readers to `metadata` which will have its own examples?

Reply:
Good question. My intention was to provide a simple snippet people could copy-paste to do the job properly. So I'd agree to moving the example to `metadata`. I'm not entirely sure people will find it when linked from `exists` so maybe add a simple note but maybe that's excessive.